.github/workflows/ci_rosdoc_lite_check.yml

```diff
@@ -0,0 +1,16 @@
+name: CI-rosdoc-lite check
+
+on: [push, pull_request]
+
+jobs:
+  rosdoc_lite:
+    name: "rosdoc_lite check for psen_scan_v2"
+    runs-on: ubuntu-20.04
+    container: ros:noetic-ros-core
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install rosdoc-lite
+        run: sudo apt-get update && sudo apt-get install ros-noetic-rosdoc-lite graphviz -y
```

Comment:
```suggestion
        run: sudo apt-get update && sudo apt-get install ros-noetic-rosdoc-lite doxygen -y
```

Reply:
Should not be needed https://github.com/ros-infrastructure/rosdoc_lite/blob/981c151d22c567475bc0834729c9f282985c18b1/package.xml#L25